api/v1beta1/webhook_suite_test.go

```diff
@@ -97,6 +101,10 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 	Expect(k8sClient).NotTo(BeNil())
 
+	openshiftCheck, err := utils.NewOpenshiftValidator(cfg)
```

Comment:
This looks like a test of the utils package, not? It shouldn't be in BeforeSuite but in an actual test. And ideally not in this api package.

Reply:
>  It shouldn't be in BeforeSuite but in an actual test.  And ideally not in this api package.

Where would it make more sense to place it? I thought of placing it here as it would happen once per suite and Webhook unit tests and not in the [BeforeEach](https://github.com/medik8s/node-maintenance-operator/pull/110/files#diff-b1c28abc51adb423cd06fb706026df133e02e07191bdb6c184d25505aca6c30bR104-R106) of the unit tests. 
But from your answer, I understand that api package does not seem right. Do you have any other place in mind?

Reply:
- every test should be in an `It()`
- ideally unit test suites should be where the code is, so for this one in the `utils` package

Reply:
>so for this one in the utils package

Are you suggesting moving all the unit test logic and files for Webhook from _api_ package to _utils_ package?

Reply:
no, just the validator test